### DISCUSSION TITLE
[Esame 31 Agosto 2021] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#249] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-13 194415](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/c74e11ea-0d23-49f8-8a0b-8258d39c8008)

**Comment by cornflexxx** [comment]
Come trovato nell' esercizio #109  : 

$$\mathbb{P}\\{X=x| X+Y=z\\}=\frac{\mathbb{P}\\{X=x\cap Y=z-x\\}}{\mathbb{P}\\{X+Y=z\\}}=\frac{\mathbb{P}\\{X=x\\}\cdot \mathbb{P}\\{Y=z-x\\}}{\mathbb{P}\\{X+Y=z\\}}\sim Bin(z,\lambda /(\lambda+\mu))$$

Ne segue che : 

$$\mathbb{E}(X|X+Y)=z\frac{\lambda}{\lambda+\mu}$$

----------------

$$\mathbb{P}\\{X+Y=z| X=x\\}=\frac{\mathbb{P}\\{Y=z-x\\}\cdot\mathbb{P}\\{ X=x\\}}{\mathbb{P}\\{X=x\\}}=\mathbb{P}\\{Y=z-x\\}$$

Quindi  essendo comunque una distribuzione di Poisson il valore atteso è:

$$\mathbb{E}(X+Y|X)=\sum^{+\infty}_z  z\cdot \mathbb{P}\\{Y=z-x\\} =e^{-\mu}\sum^{+\infty} z\cdot \frac{\mu^{z-x}}{(z-x)!}=e^{-\mu}\sum^{+\infty} (z-x+x)\cdot \frac{\mu^{z-x}}{(z-x)!}=\cdots$$

$$\cdots=e^{-\mu}\sum^{+\infty} (z-x)\cdot \frac{\mu^{z-x}}{(z-x)!}+xe^{-\mu}\sum^{+\infty} \frac{\mu^{z-x}}{(z-x)!}=\mu + x$$

Non sono certo che abbia senso il risultato dato che comunque dovrebbe essere il valore atteso di una dist. di Poisson quindi $\mu$

**Comment by Elia-Belli** [comment]
> Non sono certo che abbia senso il risultato dato che comunque dovrebbe essere il valore atteso di una dist. di Poisson quindi μ

@cornflexxx in realtà mi torna, perchè fissato $X=x$ (prob condizionata) al variare di $Y$ ci sarà uno scostamento di $x$ a tutti i valori assunti da $Y$, inoltre il valore atteso è lineare quindi se $\mathbb{E}(Y)=\mu$ e fissato $X=x$ abbiamo $\mathbb{E}(X)=x$ abbiamo che:
$$\mathbb{E}(Y+X)=\mathbb{E}(Y)+\mathbb{E}(X)=\mu +x$$
